build.sbt

```diff
@@ -1,10 +1,10 @@
 import sbt.{Test, project}
 import wartremover.Wart
 
-//val scala3 = "3.2.0"
+val scala3 = "3.4.2"
```

Comment:
found that 3.3 has an issue with scalatest unused unit warnings https://github.com/scala/scala3/issues/18804